packages/frontend/@n8n/i18n/src/locales/en.json

```diff
@@ -2002,6 +2002,8 @@
 	"settings.log-streaming.eventGroup.n8n.user": "User",
 	"settings.log-streaming.eventGroup.n8n.node": "Node Executions",
 	"settings.log-streaming.eventGroup.n8n.node.info": "Will send step-wise execution events every time a node executes. Please note that this can lead to a high frequency of logged events and is probably not suitable for general use.",
+	"settings.log-streaming.eventGroup.n8n.runner": "Runner tasks",
```

Comment:
I don't see those being used anywhere. Is this in anticipation to new UI options ?

Reply:
You'll find those [here](https://share.cleanshot.com/r3khysL2). Failure to add them will show the raw i18n key e.g. `settings.log-streaming.eventGroup.n8n.audit`